### PR TITLE
Changes by vikram

### PR DESCRIPTION
**Pull request by VIKRAMSUBRAMANIAN**

### CODE REVIEW

File: pip.txt [hunk 1]
```diff
@@ -11,7 +11,7 @@ pip install mysqlclient
 (sudo apt-get install python-dev python3-dev
 sudo apt-get install libmysqlclient-dev)
 
-pip install flask-migrate
+pip install flask-migratemysql_secure_installation
```
Comment (Laksh47) [inline pip.txt]: This should be in the next line

File: config.py [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 class Config(object):
   SQLALCHEMY_DATABASE_URI = os.environ.get('DATABASE_URL') or \
-    'mysql://root:@localhost/segarage'
+    'mysql://root:hello123@localhost/segarage'
```
Comment (Laksh47) [inline config.py]: These changes need not be committed

File: app/templates/specific_paper.html [hunk 3]
```diff
@@ -133,7 +141,8 @@ <h3>{{ paper.paper_name }}</h3>
   <div class="modal_content">
     <span class="close">&times;</span>
     <div class="modal_msg">
-      Please do not use this feature if you are not the contact author, only the contact authors get the link to upload the tools!
+      Please do not use this feature if you are not the contact author, only the contact authors get the link to upload
+      the tools!
```
Comment (Laksh47) [inline app/templates/specific_paper.html]: sure it should be in the new line?

File: app/templates/specific_paper.html [hunk 4]
```diff
@@ -55,66 +55,74 @@ <h3>{{ paper.paper_name }}</h3>
...
-      {% for file in paper.files %}
-        {% if file != 'None' %}
+
+      </td>
+    </tr>
+    <tr>
+      <td>Tags:</td>
+      <td>
+
+        {{ search_tags }}
```
Comment (Laksh47) [inline app/templates/specific_paper.html]: right now this will be null, right?

File: app/templates/specific_paper.html (hunk 3 above)
Comment (VIKRAMSUBRAMANIAN) [reply app/templates/specific_paper.html]: Looks like a mistake

File: app/templates/specific_paper.html (hunk 4 above)
Comment (VIKRAMSUBRAMANIAN) [reply app/templates/specific_paper.html]: yes

**Pull request merged**